PowerUp/app/src/main/res/layout-land/game_activity.xml

```diff
@@ -52,32 +51,33 @@
 
     <ImageView
         android:id="@+id/askerImageView"
-        android:layout_width="180dp"
-        android:layout_height="300dp"
-        android:src="@drawable/marco"
+        android:layout_width="@dimen/gameActivity_iv_width"
+        android:layout_height="@dimen/gameActivity_iv_height"
         android:layout_above="@+id/power_bar"
+        android:layout_alignParentEnd="true"
         android:layout_alignParentRight="true"
-        android:layout_alignParentEnd="true" />
+        android:src="@drawable/marco" />
 
 
     <TextView
         android:id="@+id/questionView"
-        android:hint="Question"
-        android:layout_width="400dp"
+        android:layout_width="@dimen/questionview_tv_width"
         android:layout_height="wrap_content"
+        android:layout_below="@+id/scenarioNameEditText"
+        android:layout_centerHorizontal="true"
         android:background="@android:color/transparent"
         android:ems="@integer/text_view_ems"
+        android:hint="@string/question"
         android:textColor="#4e4d40"
-        android:textSize="@dimen/game_text_size"
-        android:layout_below="@+id/scenarioNameEditText"
-        android:layout_centerHorizontal="true" />
+        android:textSize="@dimen/game_text_size" />
 
     <RelativeLayout
+        android:id="@+id/relativeLayout"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_below="@+id/scenarioNameEditText"
-        android:layout_marginTop="10dp"
-        android:id="@+id/relativeLayout">
+        android:layout_marginTop="@dimen/gameActivity_rl_marginTop">
+
```

Comment:
Change all the id's related to face/hair/cloth/eye to the common id name used for inflation in UiUtils class else it will not be able to inflate and is giving a runtime error.